### PR TITLE
Add livepatch to manager instead of monitor, refactored

### PR DESCRIPTION
**Pull request by silverdrake11**
Manual Testing Instructions:

This will require a client with this code and a server with the linked below code running so that you can see the new messages.

1) Check out the server branch https://github.com/canonical/landscape-server/pull/82 and run server
2) Start server
3) Change the code in client under landscape/client/manager, the files for livepatch, ubuntu pro, and pro reboot. Change the code so that it runs every 60 seconds so you can see it
4) (Optional) Comment out some of the other noisy plugins in landscape/client/monitor/config.py and landscape/client/manager/config.py to make logs cleaner
5) Register client to the server and accept the computer
6) Monitor the client logging to see messages are getting sent and cached from ubuntu pro, reboot required, and livepatch
7) Send api requests to` http GET "localhost:9091/v2/computer/ID" Authorization:"Bearer $JWT"` so that you can see the new messages

Note a separate endpoint will be created to clean up the output. This is just testing that the data is going through

### CODE REVIEW

**Comment by silverdrake11** [comment]
Note this is still drafting because to test, it requires landscapes server changes

**Review by srunde3 — CHANGES_REQUESTED** [review]
In addition to inline comments:

You may want these plugins included in the defaults for the landscape client snap.  See `snap/hooks/default-configure`

**Comment by silverdrake11** [comment]
> In addition to inline comments:
> 
> You may want these plugins included in the defaults for the landscape client snap. See `snap/hooks/default-configure`

I'll just add UbuntuProInfo for now. Which is strange that it's not there since there was this pull request recently to enable it for the snap https://github.com/canonical/landscape-client/commit/b2ce9c851cb6fecff945d286a5ca685cbaeca74e

File: landscape/client/manager/tests/test_livepatch.py [hunk 1]
```diff
@@ -28,11 +28,11 @@ def setUp(self):
     def test_livepatch(self):
         """Tests calling livepatch status."""
         plugin = LivePatch()
-        self.monitor.add(plugin)
 
         with mock.patch("subprocess.run") as run_mock:
             run_mock.side_effect = subprocess_livepatch_mock
-            plugin.exchange()
+            self.monitor.add(plugin)
```
[the remark this answers is not in the document]
Comment (silverdrake11) [reply landscape/client/manager/tests/test_livepatch.py]: Ok good point I updated

**Review by srunde3 — APPROVED** [review]
LGTM :ship: :hippopotamus:

**Pull request merged**